app/services/provider_interface/hesa_data_export.rb

```diff
@@ -38,9 +36,9 @@ def export_row(application_choice)
         'DEGTYPE' => pad_hesa_value(application.first_degree, :qualification_type_hesa_code, 3),
         'DEGSBJ' => pad_hesa_value(application.first_degree, :subject_hesa_code, 4),
         'DEGCLSS' => pad_hesa_value(application.first_degree, :grade_hesa_code, 2),
-        'institution_country' => application.first_degree.institution_country,
-        'DEGSTDT' => first_degree_start,
-        'DEGENDDT' => first_degree_end,
+        'institution_country' => application.first_degree.present? ? application.first_degree.institution_country : NO_DEGREE,
```

Comment:
```suggestion
        'institution_country' => application.degrees? ? application.first_degree.institution_country : NO_DEGREE,
```
Can we use the new `#degrees?` method here?

Reply:
Good catch, I did this before adding the method! I will do it!